src/clients/components/MainWrapper.tsx

```diff
@@ -7,12 +7,13 @@ const useStyles = createStyles((theme) => ({
     backgroundColor:
       theme.colorScheme === "dark" ? theme.colors.blue[3] : theme.white,
     color: theme.colorScheme === "dark" ? theme.colors.dark[0] : theme.black,
+    height: "calc(100vh - 60px)",
   },
 }));
 
 const MainWrapper = ({ children }: { children: React.ReactNode }) => {
   const { classes } = useStyles();
-  return <div className={classes.root}>{children}</div>;
+  return <main className={classes.root}>{children}</main>;
 };
```

Comment:
@devjoylee `src/app/page.tsx` also has `main` tag (L9). Probably should we also fix `src/app/page.tsx` L9 (from `main` to `div`)?